docs/current_docs/quickstart/853930-directories.mdx

```diff
@@ -43,86 +47,47 @@ Dagger's core types (`Container`, `Directory`, `Service`, `Secret`, ...) are all
 
 When a function returns a core type - for example, a directory - the caller typically continues the chain by calling a function from that directory - for example, export it to the local filesystem, modify it, mount it into a container, etc.
 
-### Export the build directory
+### Export the built binary
 
 Using this knowledge of function chaining, call the builder function again and continue the chain by interacting with the resulting directory.
 
 Begin by listing the directory's contents:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project https://github.com/dagger/dagger --args ./cmd/dagger entries
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.10 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger --os=$(uname -s | awk '{print tolower($0)}') entries
 ```
 
 The result should look like this:
 
-```shell
-.changes
-.changie.yaml
-.dockerignore
-.git
-.github
-.gitignore
-.golangci.yml
-.goreleaser.common.yml
-.goreleaser.nightly.yml
-.goreleaser.yml
-.markdownlint.yaml
-.readthedocs.yaml
-CHANGELOG.md
-CODE_OF_CONDUCT.md
-CONTRIBUTING.md
-LICENSE
-NOTICE
-README.md
-RELEASING.md
-auth
-ci
-cmd
-core
+```
 dagger
-dagql
-docs
-engine
-examples
-go.mod
-go.sum
-hack
-helm
-install.sh
-internal
-network
-sdk
-telemetry
-tracing
-zenith
 ```
 
-This does look like the contents of the Dagger repository after build. Great! Next, export its contents to the local filesystem:
+This does look like a compiled binary. Great!
+
+Next, export the directory to the host filesystem. You can do this by using the global `--output` flag to export the directory to a custom path (`./my-dagger-build`) on the host:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.5 call build --project https://github.com/dagger/dagger --args ./cmd/dagger export --path ./my-dagger-build
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.1.10 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger --os=$(uname -s | awk '{print tolower($0)}') --output=./my-dagger-build
```

Comment:
Can we use the shorthand `-o` instead? I get it that `--output` is more verbose and descriptive, which is good in docs, but people tend to copy. The `-o` shorthand is very commonly used for `--output` in CLIs, and we already use `-m` everywhere which contrary to `-o`, is specific to dagger.

Reply:
Modified